### ISSUE TITLE
Add notes in GitHub releases for each new release

### ISSUE THREAD

**Issue by mahozad** [square/okio#1066]
Could you please create a new release notes in [the repository releases page] for each new version?

This allows for watching just the releases notifications of the repository and be notified of new releases and their changelog.

[This page] and [this issue] are related.

---

As a side note, I have written a [GitHub workflow job] for [one of my own libraries] that automatically generates the release notes from the repository CHANGELOG.md file (and also creates the showcase APK) so I do not have to create the release notes for GitHub releases again and by myself.

Also, as you probably know, there are ready-to-use GitHub actions like [this] for that purpose.

  [the repository releases page]: https://github.com/square/okio/releases
  [This page]: https://github.blog/changelog/2018-11-27-watch-releases/
  [this issue]: https://github.com/isaacs/github/issues/410
  [GitHub workflow job]: https://github.com/mahozad/android-pie-chart/blob/master/.github/workflows/publish-new-release.yml#L199
  [one of my own libraries]: https://github.com/mahozad/android-pie-chart/releases
  [this]: https://github.com/marketplace/actions/release-changelog-builder

**Comment by JakeWharton** [comment]
Yes we have the infrastructure in other repositories of ours to copy from. This repo is just much older and needs updated.
